build.gradle

```diff
@@ -223,8 +223,8 @@ configurations {
 }
 
 // We want no transitive dependencies
-configurations.all {
-    transitive = false;
+configurations.configureEach {
```

Comment:
Configuration avoidance. It's not related to the fix. I have unadvertendly pulled it in. Could be left off for commit coherence.